.github/workflows/focus-ios-ui-tests.yml

```diff
@@ -66,6 +66,10 @@ jobs:
               run: |
                 gem install xcpretty -v 0.3.0
                 pip install blockkit==1.9.1
+                # Reduce network flakiness
+                sudo sysctl -w net.link.generic.system.hwcksum_tx=0
+                sudo sysctl -w net.link.generic.system.hwcksum_rx=0
```

Comment:
Not sure if this is needed. https://github.com/actions/runner-images/issues/1187